tests/utils/test_model_debugging_utils.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# Copyright 2025 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import json
+import os
+import tempfile
+import unittest
+from pathlib import Path
+
+from transformers import is_torch_available, is_vision_available
+from transformers.model_debugging_utils import model_addition_debugger_context
+
+
+if is_vision_available():
+    pass
+
+if is_torch_available():
+    import torch
+    from torch import nn
+
+
+class ToyModel(nn.Module):
```

Comment:
Should be also under the `if is_torch_available()`, otherwise we don't need a guard actually